docs/deployment/manifest-format.md

```diff
@@ -384,11 +492,35 @@ Example manifest:
 {
   "resources": {
     "sql1": {
-      "type": "sqlserver.connection.v1",
-      "connectionString": "Server=myexistingserver;Database=inventory;Trusted_Connection=True;"
+      "type": "sqlserver.server.v1"
     },
     "sql2": {
-      "type": "sqlserver.server.v1"
+      "type": "container.v0",
+      "image": "mcr.microsoft.com/mssql/server:2022-latest",
+      "env": {
+        "ACCEPT_EULA": "Y",
+        "MSSQL_SA_PASSWORD": "{sqlservercontainer.inputs.password}"
+      },
+      "bindings": {
+        "tcp": {
+          "scheme": "tcp",
+          "protocol": "tcp",
+          "transport": "tcp",
+          "containerPort": 1433
+        }
+      },
+      "connectionString": "Server={sqlservercontainer.bindings.tcp.host},{sqlservercontainer.bindings.tcp.port};User ID=sa;Password={sqlservercontainer.inputs.password};TrustServerCertificate=true;",
```

Comment:
Should be `Server={sql2.bindings.tcp.host},{sql2.bindings.tcp.port};User ID=sa;Password={sql2.inputs.password};TrustServerCertificate=true;`

Reply:
Done